core/src/test/java/org/apache/accumulo/core/spi/compaction/DefaultCompactionPlannerTest.java

```diff
@@ -152,12 +171,126 @@ public void testRunningCompaction() {
     // planner should compact.
     var job = getOnlyElement(plan.getJobs());
     assertEquals(candidates, job.getFiles());
-    assertEquals(CompactionExecutorIdImpl.externalId("medium"), job.getExecutor());
+    assertEquals(CompactionExecutorIdImpl.internalId(csid, "medium"), job.getExecutor());
+  }
+
+  /**
+   * Tests that the maxOpen property overrides the deprecated open.max property with the default
+   * service
+   */
+  @Test
+  @SuppressWarnings("removal")
+  public void testOverrideMaxOpenDefaultService() {
+    // Set old property and use that for max open files.
+    ConfigurationCopy aconf = new ConfigurationCopy(Map.of(), DefaultConfiguration.getInstance());
+    aconf.set(Property.TSERV_MAJC_THREAD_MAXOPEN.getKey(), "17");
+    ConfigurationImpl config = new ConfigurationImpl(aconf);
+
+    ServiceEnvironment senv = EasyMock.createMock(ServiceEnvironment.class);
+    EasyMock.expect(senv.getConfiguration()).andReturn(config).anyTimes();
+    EasyMock.replay(senv);
+
+    // Use the CompactionServicesConfig to create options based on default property values
+    var CompactionServices = new CompactionServicesConfig(aconf, log::warn);
```

Comment:
This variable name was confusing me, I thought it was a class name later and thought something was calling static method on a class.

```suggestion
    var compactionServices = new CompactionServicesConfig(aconf, log::warn);
```

Reply:
Yeah that's an oversight. I'll fix the var naming.